amethyst_input/src/bundle.rs

```diff
@@ -111,3 +123,40 @@ where
         Ok(())
     }
 }
+
+/// An error occurred while loading the bindings file.
+#[derive(Debug)]
+pub enum BindingsFileError<AX, AC> {
+    /// Problem in amethyst_config
+    ConfigError(ConfigError),
+    /// Problem with the bindings themselves.
+    BindingError(BindingError<AX, AC>),
+}
+
+impl<AX, AC> Display for BindingsFileError<AX, AC> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
+        match self {
+            BindingsFileError::ConfigError(ref err) => Display::fmt(err, f),
```

Comment:
fun fact, you can replace `=> Display::fmt(err, f),` with `|`, and 🦀 rust will infer it from the line below

Reply:
Even if the types differ? While the literal code text might be identical these aren't the same operation.

Reply:
ah! probably not. haha